userspace/libsinsp/test/sinsp_with_test_input.h

```diff
@@ -362,12 +362,24 @@ class sinsp_with_test_input : public ::testing::Test {
 		return ret;
 	}
 
+	// Return true if `field_name` exists in the filtercheck list.
+	// The field value could also be NULL, but in this method, we are not interested in the value.
```

Comment:
Do we ever need to check for this, or are you thinking about tests that need it? Honest question!

Reply:
Unless there is something that expect the `throw()` somewhere and now could use `field_exists` (not sure)

Reply:
yes, there are cases in some plugin tests that I want to write that need this new `field_exists` helper. It's true that I can use something like `ASSERT_NO_THROW` but this is not so user friendly IMO

Reply:
Moreover `field_has_value` is maybe more meaningful, WDYT?

Reply:
> plugin tests
That's a great use case. Thanks for the explanation!